taichi/backends/device.h

```diff
@@ -397,13 +397,26 @@ inline bool operator&(AllocUsage a, AllocUsage b) {
   return static_cast<int>(a) & static_cast<int>(b);
 }
 
+class StreamSemaphoreObject {
```

Comment:
Why Stream in this naming? (Initially I thought we had a stream of semaphores..)

Reply:
Just that on Windows Semaphore is a system type, will conflict with Win32 API